packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -475,6 +483,10 @@ export const BRIDGE_ENCODERS: {
     ]),
     [ERC20BridgeSource.KyberDmm]: AbiEncoder.create('(address,address[],address[])'),
     [ERC20BridgeSource.Lido]: AbiEncoder.create('(address)'),
+    [ERC20BridgeSource.Firebird]: AbiEncoder.create([
+        { name: 'router', type: 'address' },
+        { name: 'pool', type: 'address' },
+    ]),
```

Comment:
From my previous comment this would be 
```suggestion
    [ERC20BridgeSource.Firebird]: AbiEncoder.create([
        { name: 'router', type: 'address' },
        { name: 'pools', type: 'address[]' },
    ]),
```

Reply:
handled in 25434c0